generators/component/templates/custom_style.css

```diff
@@ -0,0 +1,4 @@
+/**
+  * Custom CSS file for all the components
```

Comment:
This is essentially not a custom css file. This is the main bootstrap file where one would import all the components. Could you update? @afamyial